packages/editor/src/components/rich-text/index.js

```diff
@@ -611,27 +612,32 @@ export class RichText extends Component {
 			}
 
 			if ( this.props.multiline ) {
-				if ( ! this.props.onSplit ) {
-					return;
-				}
-
 				const record = this.getRecord();
 
-				if ( ! isEmptyLine( record ) ) {
-					return;
+				if ( this.props.onSplit && isEmptyLine( record ) ) {
+					this.props.onSplit( ...split( record ).map( this.valueToFormat ) );
+				} else {
+					this.onChange( insert( record, '\u2028' ) );
```

Comment:
Why Unicode 2028 instead of a plain (ASCII-level) line break?

Reply:
Because we're using that to delimit lines in multiline values. We could probably benefit from a helper or a constant...

Reply:
And `\n` are used for single line breaks which convert to `<br>` elements.

Reply:
That's fair, but let's add a comment for it.

Reply:
Comment then 🚢 🚢 !